src/Sulu/Bundle/ResourceBundle/Resources/public/js/components/condition-selection/main.js

```diff
@@ -229,6 +229,79 @@ define([], function() {
         },
 
         /**
+         * Removes a field from the used fields and adds it to the available fields
+         * should always be called in context with update select fields
+         * @param fieldName
+         */
+        removesUsedField = function(fieldName) {
+            var value;
+
+            if (!!this.usedFields[fieldName]) {
+                value = this.usedFields[fieldName];
+                delete this.usedFields[fieldName];
```

Comment:
I think the `delete` keyword is just for deleting properties, not array elements. I guess this is not working as intended.